Makefile

```diff
@@ -1,21 +1,46 @@
 all: markdown manpages
 
-INSTALL_PREFIX:=/usr/local/bin
+INSTALL_PREFIX:=/usr/local
 
-$(INSTALL_PREFIX)/%: %
-	install -Dm755 $< $@
+install: install-bin install-man
+
+clean:
+	rm -f $(addprefix man/, $(addsuffix .8.gz, $(call ls_bin)))
+	[ ! -d man ] || rmdir man
+	for btrfs_mnt in btrfs.working.mnt btrfs-snapshots.working.mnt; do \
+		! mountpoint -q "/tmp/snazzer-tests/$$btrfs_mnt" ||            \
+		umount "/tmp/snazzer-tests/$$btrfs_mnt";                       \
+	done
+	rm -rf /tmp/snazzer-tests
+
+distclean: clean
+	rm -f bats
+	rm -rf tmp/bats
+	[ ! -d tmp ] || rmdir tmp
+
+test: bats-tests prune-tests
 
-install: $(shell find . -maxdepth 1 -executable -type f \
-	-printf '$(INSTALL_PREFIX)/%p\n' )
+ls_bin = $(shell find . -maxdepth 1 -executable -type f -printf '%P\n')
 
 uninstall:
-	rm $(shell find . -maxdepth 1 -executable -type f \
-	   	-printf '$(INSTALL_PREFIX)/%p\n')
+	rm -f $(addprefix $(INSTALL_PREFIX)/bin/, $(call ls_bin))
+	rm -f $(addprefix $(INSTALL_PREFIX)/share/man/man8/, $(addsuffix .8.gz, \
+		$(call ls_bin)))
 
-clean:
-	rm -r /tmp/snazzer-tests
+install-bin: $(addprefix $(INSTALL_PREFIX)/bin/, $(call ls_bin))
 
-test: bats-tests prune-tests
+$(INSTALL_PREFIX)/bin/%: %
+	install -Dm755 $< $@
+
+install-man: $(addprefix $(INSTALL_PREFIX)/share/man/man8/, $(addsuffix .8.gz, \
+	$(call ls_bin)))
+
+$(INSTALL_PREFIX)/share/man/man8/%.8.gz: man/%.8.gz | \
+	$(INSTALL_PREFIX)/share/man/man8
+	install -Dm644 $< $@
+
+$(INSTALL_PREFIX)/share/man/man8:
+	mkdir $(INSTALL_PREFIX)/share/man/man8
```

Comment:
This line broke in my tests, as I had no `/usr/local/share/man` directory. Initially I would have gone with a `mkdir -p`, but I'm quite sure this target can actually be left out; see other comment.